charts/cf-runtime/Chart.yaml

```diff
@@ -14,10 +14,8 @@ maintainers:
     url: https://codefresh-io.github.io/
 annotations:
   artifacthub.io/changes: |
-    - kind: changed
-      description: Add option to provide labels to dind and engine pods
-    - kind: fixed
-      description: Fix Offline-Logging Redis connection
+    - kind: added
+      description: Add support for `strict_fail_fast` field in pipelines
```

Comment:
Can you remove backticks here? ArtifactHub probably won't like in the changes annotation.

Reply:
✅